src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -436,7 +445,16 @@ Exception onConferenceCallStarted(Call jvbConferenceCall)
 
         if (destination == null)
         {
-            CallManager.acceptCall(sipCall);
+            try
+            {
+                CallManager.acceptCall(sipCall);
+            }
+            catch(OperationFailedException e)
+            {
+                hangUpSipCall(e.getErrorCode(), "Cannot answer call");
```

Comment:
Do we want to include the meessage as well (`e.getMessage()`)?

Reply:
Well, that ends up in the sip message ... I was thinking it maybe too long ...